src/ApiVersion.Api/DTOs/AspApiVersions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections;
+
+namespace ApiVersion.Api.DTOs;
+
+internal class AspApiVersions : IList<AspApiVersion>
```

Comment:
```suggestion
internal class AspApiVersions : List<AspApiVersion>
```

This is mostly pass through. In that case we can inherit `List` instead of implementing `IList`.

Reply:
Done